lib/guard/standardrb.rb

```diff
@@ -4,15 +4,21 @@
 
 module Guard
   class Standardrb < Plugin
-    def initialize(options = {})
+    attr_reader :fix
+
+    def initialize(opts = {})
       super
-      @options = options
+      @fix = opts[:fix]
     end
 
     def start
+      Guard::Compat::UI.info "Standardrb --fix = #{fix}"
       Guard::Compat::UI.info "Inspecting Ruby code style of all files with standardrb"
     end
 
+    def stop
```

Comment:
I don't see where this method is referred to at all. Is it needed?

Reply:
nop you are right. I used it in a test then removed it 👍